src/HtmlUtils.tsx

```diff
@@ -523,7 +527,7 @@ export function linkifyElement(element: HTMLElement, options = linkifyMatrix.opt
  * @param {object} [options] Options for linkifyString. Default: linkifyMatrix.options
  * @returns {string}
  */
```

Comment:
the linkifyString/Element functions above could do with return types while you're in the area.

maybe some brief doc fixes too...